client/src/js/partialuserforms/partialsubmission.js

```diff
@@ -22,9 +22,13 @@ const getElementValue = (element, fieldName) => {
       return valueArray;
     }
     return '';
+  } else if (element.getAttribute('type') === 'file') {
+    return element.files[0];
```

Comment:
Can you check if files is not empty, otherwise it will return undefined value

Reply:
The javascript itself doesn't seem to really care about it, but yeps :)